components/console-backend-service/internal/domain/k8s/resourcequota_service.go

```diff
@@ -46,6 +47,28 @@ func (svc *resourceQuotaService) ListResourceQuotas(namespace string) ([]*v1.Res
 	return result, nil
 }
 
+func (svc *resourceQuotaService) CreateResourceQuota(namespace string, name string, memoryLimits string, memoryRequests string) (*v1.ResourceQuota, error) {
+	ResourceQuota := &v1.ResourceQuota{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: v1.ResourceQuotaSpec{
+			Hard: v1.ResourceList{
+				v1.ResourceLimitsMemory:   resource.MustParse(memoryLimits),
```

Comment:
If the wrong value is passed to `MustParse`, the client will see only cryptic "internal server error". I think it would be better to use `ParseQuantity` here and return the error.

Reply:
Also I prefer use this function before creating rq object, and check error, also for `memoryRequests` value, like:

``` go
memoryLimitsParsed, err := resource.ParseQuantity(memoryLimits)
if err != nil {
   ... logic for error
}
...
```